styles/components-styles/footer.css

```diff
@@ -0,0 +1,53 @@
+.connect {
+  min-height: 30vh;
+  color: white;
+  background: var(--COLOR-PRIMARY-XXDARK);
+  text-align: center;
+}
+
+.contact-content {
+  justify-content: space-between;
+  align-items: start;
+}
+
+.footer-item {
+  align-items: start;
+}
+
+.footer-item a:hover {
+  color: var(--COLOR-COMPLEMENTARY-LIGHT);
+}
+
+.email-input {
+  border-radius: 4px 0 0 4px;
+}
+
+.footer-icon:hover {
+  transition: all 0.2s ease;
+}
+
+.footer-icon:hover {
```

Comment:
both are targeting the same element can merge both

Reply:
yes, refactored this part. Thanks! Didn't notice that I mentioned `:hover` twice.